.github/workflows/update-snapshots.yml

```diff
@@ -6,17 +6,33 @@ jobs:
   update-snapshots:
     runs-on: ubuntu-latest
     if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/update-snapshots') }}
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
     steps:
       - name: Generate token
         uses: tibdex/github-app-token@v1
         id: generate-token
         with:
           app_id: ${{ secrets.TOKEN_APP_ID }}
           private_key: ${{ secrets.TOKEN_APP_PRIVATE_KEY }}
+      - name: Add seen reaction
+        uses: peter-evans/create-or-update-comment@v3
```

Comment:
does this work from forks?

Reply:
ahh, nvm, the event is for the created comment so it should work alright

Reply:
I think so, it's just using a comment ID? I'll definitely test it -- I always use a fork here